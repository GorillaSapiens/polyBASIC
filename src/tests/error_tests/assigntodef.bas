def a(x) x + 1
a(5) = 2
