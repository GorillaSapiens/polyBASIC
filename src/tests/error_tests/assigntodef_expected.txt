ERROR: @2:16, CANNOT ASSIGN TO DEFINITION ❮a❯
